src/CoreBundle/Migrations/Schema/V200/Version20240709222600.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+/* For licensing terms, see /license.txt */
+
+namespace Chamilo\CoreBundle\Migrations\Schema\V200;
+
+use Chamilo\CoreBundle\Migrations\AbstractMigrationChamilo;
+use Doctrine\DBAL\Schema\Schema;
+
+final class Version20240709222600 extends AbstractMigrationChamilo
+{
+    public function getDescription(): string
+    {
+        return 'Create permissions and permission_rel_roles tables';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('
+            CREATE TABLE IF NOT EXISTS permissions (
```

Comment:
Tables in Chamilo have to be named in singular (`permission`, not `permissions`). This is an old standard that we try to maintain in all contexts except when it would bring confusion vs something else.